examples/experimental/tongye/opacus_with_Plan/Opacus_Syft_Data_Scientist.ipynb

```diff
@@ -0,0 +1,627 @@
+{
```

Comment:
<p>But this is the pointer to the <strong>Dataset</strong>, not the dataloader. This length is the total number of examples, whereas if we check the length of the dataloader, we'd get the total number of batches.</p><p> </p><p>If we want to pass a pointer to the dataset instead that is totally fine, but then we should change the text cells to reflect that it's not in fact a dataloader :) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/5333/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>right, it's a typo, I did want length of dataloader, it should be <code>dl_length=train_loader_ptr.<strong>len</strong>()</code></p>